eeg_cofitage/participants.json

```diff
@@ -1,14 +1,17 @@
 {
+  "participant_id":{
+    "LongName":"Participant Id",
+    "Description":"label identifying a particular subject"
+  },
   "age":{
     "LongName":"Age",
     "Description":"Age of a subject",
     "Units":"year"
   },
   "sex":{
     "LongName":"Sex",
-    "Description":"Sex of a subject",
+    "Description":"Gender of a subject",
```

Comment:
you probably mean `sex`, i.e., the biological distinction between male and female, see: https://en.wikipedia.org/wiki/Sex_and_gender_distinction

Reply:
Well, I learned something today. Fixed.